test/ranges_test.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'test_helper'
```

Comment:
Minor note: it is a convention to name the test file for a class the same name as the lib file with an `_test`